stock_picking_operations_multilang/tests/test_stock_picking_operations_multilang.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+class TestPickingOperationsLanguage(BaseCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        ja = (
+            cls.env["res.lang"]
+            .with_context(active_test=False)
+            .search([("code", "=", "ja_JP")])
+        )
+        cls.env["base.language.install"].create({"lang_ids": ja.ids}).lang_install()
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Test Partner", "lang": "ja_JP"}
+        )
+        cls.picking_type_in = cls.env.ref("stock.picking_type_in")
+        cls.picking = cls.env["stock.picking"].create(
+            {
+                "partner_id": cls.partner.id,
+                "picking_type_id": cls.picking_type_in.id,
+            }
+        )
+        cls.warehouse = cls.picking_type_in.warehouse_id
+        cls.report = cls.env.ref("stock.action_report_picking")
+
+    def test_stock_picking_operations_language(self):
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="en-US"', str(report_result[0]))
+
+        self.warehouse.picking_operation_language_option = "partner"
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="ja-JP"', str(report_result[0]))
+
+        self.partner.lang = "en_US"
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="en-US"', str(report_result[0]))
+
+        self.picking.partner_id = False
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="en-US"', str(report_result[0]))
+
+        self.warehouse.picking_operation_language_option = "warehouse"
+        self.warehouse.warehouse_language = "ja_JP"
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="ja-JP"', str(report_result[0]))
+
+        self.warehouse.warehouse_language = "en_US"
+        report_result = self.report._render_qweb_pdf(self.report.id, [self.picking.id])
+        self.assertIn('lang="en-US"', str(report_result[0]))
+
+    def test_active_lang_count(self):
+        """Test computation of active_lang_count"""
+        self.assertGreater(self.warehouse.active_lang_count, 0)
+
+        # Simulate change in warehouse_language
+        self.warehouse.warehouse_language = "ja_JP"
+        self.warehouse._compute_active_lang_count()
+
+        # Ensure the computed count remains correct
+        self.assertEqual(
+            self.warehouse.active_lang_count, len(self.env["res.lang"].get_installed())
+        )
```

Comment:
I think this test is either unnecessary or has to be updated with proper context. In fact, I forget why we added the `@api.depends("warehouse_language")` decorator to `_compute_active_lang_count()`. Could you please check if there is an usability issue if we remove it?